src/tests/components/LocationDetails.test.js

```diff
@@ -1,13 +0,0 @@
-import React from "react";
-import { render } from "@testing-library/react";
-import LocationDetails from "../../components/LocationDetails";
-
-describe("LocationDetails", () => {
-  it("renders the correct city and location props", () => {
-    const { getByText } = render(
-      <LocationDetails city="Manchester" country="UK" />
-    );
-
-    expect(getByText("Manchester, UK")).toBeInstanceOf(HTMLHeadingElement);
```

Comment:
Again, you are only testing that there is a h1. I would check that the content of your h1 displays the city and country for the props you've passed in